### PR TITLE
Update documentation for @ExceptionHandler

### PR DESCRIPTION
**Pull request by evyrosseels**
## Description

Clarified usage of `@ExceptionHandler`, including example of usage with an interface.

## Motivation and Context

The current documentation is not very clear on how to use the exception handlers.

### CODE REVIEW

File: documentation/Extension_Point_WebScripts.md [hunk 1]
```diff
@@ -198,6 +198,18 @@ protected void handleIllegalArgument(IllegalArgumentException exception, WebScri
...
+It is also possible to separate out exception handler methods into their own class, using an interface with default methods.
+The interface can then be implemented by any WebScript that requires it:
+```java
+public interface ExceptionHandlers {
+    @ExceptionHandler(IllegalArgumentException.class)
+    default void handleIllegalArgument(IllegalArgumentException exception, WebScriptResponse response) {
+        response.setStatus(400);
+    }
+}
+```
```
Comment (vierbergenlars) [inline documentation/Extension_Point_WebScripts.md:212]: Can you add an example of the webscript class too?

File: documentation/Extension_Point_WebScripts.md [hunk 2]
```diff
@@ -198,6 +198,18 @@ protected void handleIllegalArgument(IllegalArgumentException exception, WebScri
     response.setStatus(400);
 }
 ```
+This handler method must be defined in the same WebScript that will throw the Exceptions to be handled.
```
Comment (kerkhofsd) [inline documentation/Extension_Point_WebScripts.md:201]: I would rephrase it a bit to use the correct terminology: 
`... in the same @WebScript annotated class ...`
?

However if it's more clear like it is now, it's good for me. Other opinions welcome.

Comment (evyrosseels) [reply documentation/Extension_Point_WebScripts.md:201]: I'd say your suggestion is more clear, agreed.

**Review by kerkhofsd — APPROVED** [review]

**Review by vierbergenlars — APPROVED** [review]
